conf/modules.config

```diff
@@ -217,6 +217,44 @@ process {
         ]
     }
 
+    withName: 'FLYE' {
+        memory = {
+            def assembly_memory = meta.assembly_memory ?: params.assembly_memory;
+            check_max( assembly_memory.GB + ( assembly_memory.GB * 0.5 * ( task.attempt - 1 ) ), 'memory')
+        }
+        cpus   = { check_max( 12    * task.attempt, 'cpus'    ) }
+        time   = { check_max( 16.h  * task.attempt, 'time'    ) }
+
+        ext.args = '--meta'
+    }
+
+    withName: 'RACON' {
+        cpus   = { check_max( 4                  , 'cpus'    ) }
+        memory = { check_max( 3.GB * task.attempt, 'memory'  ) }
+        time   = { check_max( 4.h  * task.attempt, 'time'    ) }
```

Comment:
I think you can simplify a lot of this modules.config file by instead using labels for to assign resources (unless you one of your modules really needs special resource allocation. 

For example the `main.nf` for `RACON` has the `label 'process_high'` line. What this is meant to refer to is a standard nf-core `base.config` file, that looks similar to the one in the [amplicon pipeline](https://github.com/EBI-Metagenomics/amplicon-pipeline/blob/main/conf/base.config).

By having a couple of labels like these that stand for specific resource allocations, it's more standard and it's easier as if many modules will need let's say `process_low` for resources you only ever have to declare it once.

Reply:
Agreed... this applies to most of other pipelines. We should do a CONFIGURATHON to adjust them